pkg/query/querier.go

```diff
@@ -75,25 +76,34 @@ func optsFromContext(ctx context.Context) *opts {
 }
 
 // StoreInfo holds meta information about a store used by query.
-type StoreInfo interface {
+type StoreInfo struct {
+	Addr string
+
 	// Client to access the store.
-	Client() storepb.StoreClient
+	Client storepb.StoreClient
+
+	// Labels that apply to all date exposed by the backing store.
+	Labels []storepb.Label
 
-	// Labels returns store labels that should be appended to every metric returned by this store.
-	Labels() []storepb.Label
+	// Minimum and maximum time range of data in the store.
+	MinTime, MaxTime int64
+}
```

Comment:
Having this as an interface for a bunch of getters ultimately just seemed like a bunch of pointless indrection. Just moved it to a struct. The cluster package depends on `pkg/query` either way.

Reply:
ok